### PR TITLE
TST: parametrized test_datasets.py tests

### PR DESCRIPTION
**Pull request by cr458**
Parametrized test for test_dataset.py
Related to: #644

### CODE REVIEW

File: geopandas/tests/test_datasets.py [hunk 1]
```diff
@@ -1,18 +1,13 @@
...
-        gdf = read_file(get_path('naturalearth_lowres'))
-        assert isinstance(gdf, GeoDataFrame)
-
-        gdf = read_file(get_path('naturalearth_cities'))
-        assert isinstance(gdf, GeoDataFrame)
-
-        gdf = read_file(get_path('nybb'))
-        assert isinstance(gdf, GeoDataFrame)
+@pytest.mark.parametrize("test_gdf",
+                         [read_file(get_path('naturalearth_lowres')),
```
Comment (jorisvandenbossche) [inline geopandas/tests/test_datasets.py]: Can you put the 'read_file(get_path' part inside the test? (so only parametrize the names of the datasets)

Comment (cr458) [reply geopandas/tests/test_datasets.py]: Yup! Fixed.

**Comment by cr458** [comment]
@jorisvandenbossche I could get started on other tests if you'd like?

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**